internal/pkg/manifest/lb_web_svc.go

```diff
@@ -165,12 +162,12 @@ func (s *LoadBalancedWebService) BuildRequired() (bool, error) {
 	return requiresBuild(s.ImageConfig.Image)
 }
 
-// TaskPlatform returns the os/arch for the service.
+// TaskPlatform returns the platform for the service.
 func (t *TaskConfig) TaskPlatform() (*string, error) {
-	if err := exec.ValidatePlatform(t.Platform); err != nil {
-		return nil, fmt.Errorf("validate platform: %w", err)
+	if t.Platform == nil {
+		return nil, nil
 	}
-	return t.Platform, nil
+	return t.Platform.PlatformString, nil
```

Comment:
Maybe in a follow-up PR:   
What will happen if a user writes:
```yaml
platform:
  osfamily: linux
  arch: x86_64
```
Right now if they're on an ARM machine we won't build the image with `--platform linux/x86_64`.

Should we instead have something like this:
```go
func (p *PlatformArgsOrString) ShortString() *string {
  if p.PlatformString != nil { return p.PlatformString }
  if p.PlatformArgs != nil && p.PlatformArgs.isValid() /* only linux and x86_64 for now */ {
     return aws.String(fmt.Sprintf("%s/%s", p.PlatformArgs.OS, p.PlatformArgs.Arch)
  }
  return nil
}
```